build_scripts/tests_runner.py

```diff
@@ -135,7 +128,50 @@ def _copy(self):
         self._log.info('-' * 50)
         self._log.info("COPYING")
 
-        # TODO: Save results on share
+        repo = self._component.trigger_repository
+        branch = repo.target_branch or repo.branch
+        build_event = self._component.build_info.build_event
+        commit_id = repo.revision
+        build_type = self._component.build_info.build_type
+        product_type = self._product_type or self._component.build_info.product_type
+        product = self._component.name
+
+        artifacts_dir = MediaSdkDirectories.get_test_dir(
+            branch=branch,
```

Comment:
Could we create dict for arguments of get_test_dir and get_test_url fuctions and use it via **?